sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests.AspNetCoreWebApp/Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.AspNetCoreWebApp.csproj

```diff
@@ -11,6 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Text.Encodings.Web" VersionOverride="4.5.1" />
```

Comment:
Why do we need to override here? Doesn't 4.7.2 also work https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L72?

Reply:
The version that was referenced automatically was 4.5.0, and guidance is to use 4.5.1 for 4.0.0 - 4.5.0. 4.7.2 will update many more references than we want.